subsys/shell/shell.c

```diff
@@ -1163,6 +1163,13 @@ static void shell_log_process(const struct shell *shell)
 		processed = shell_log_backend_process(shell->log_backend);
```

Comment:
Noticed some typos in commit "shell: Postpone enabling log backend" message. 

s/was enabled to early/was enabled **too** early/

Reply:
fixed